banking_system/user_management/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
-from .models import User, UserLogin, UserPendingApproval
+from .models import User, UserLogin, UserPendingApproval, employee_info_update
 # Register your models here.
 
 admin.site.register(User)
 admin.site.register(UserLogin)
 admin.site.register(UserPendingApproval)
+admin.site.register(employee_info_update)
```

Comment:
We should refactor the name of this when we have time

Reply:
Yes. @mayur7194  can you look into this? I am guessing you worked on this.